org.envirocar.core/src/main/java/org/envirocar/core/util/Util.java

```diff
@@ -65,27 +63,27 @@
     public static final String EXTERNAL_SUB_FOLDER = "enviroCar";
     private static ISO8601DateFormat jacksonFormat = new ISO8601DateFormat();
 
-    /**
-     * Create a file in the .enviroCar folder of the external storage.
-     *
-     * @param fileName the name of the new file
-     * @return the resulting file
-     * @throws IOException
-     */
-    public static File createFileOnExternalStorage(String fileName)
-            throws IOException {
-        File directory = resolveExternalStorageBaseFolder();
-
-        File f = new File(directory, fileName);
-        f.createNewFile();
-        if (!f.isFile()) {
-            throw new IOException(fileName + " is not a file!");
-        }
-        return f;
-    }
-
-    public static File createFileOnInternalStorage(String path, String filename) throws IOException{
-        File directory = resolveInternalStorageBaseFolder(path);
+//    /**
+//     * Create a file in the .enviroCar folder of the external storage.
+//     *
+//     * @param fileName the name of the new file
+//     * @return the resulting file
+//     * @throws IOException
+//     */
+//    public static File createFileOnExternalStorage(String fileName)
+//            throws IOException {
+//        File directory = resolveExternalStorageBaseFolder();
+//
+//        File f = new File(directory, fileName);
+//        f.createNewFile();
+//        if (!f.isFile()) {
+//            throw new IOException(fileName + " is not a file!");
+//        }
+//        return f;
+//    }
+
+    public static File createFileOnStorage(String path, String filename) throws IOException{
+        File directory = resolveStorageBaseFolder(path);
```

Comment:
Please, just refactor the method name and replace `resolveExternalStorageBaseFolder` by `resolveStorageBaseFolder` inside the existing method instead of outcommenting.